dev.yml

```diff
@@ -8,7 +8,7 @@ up:
   - custom:
       name: Minikube Cluster
       met?: test $(minikube status | grep Running | wc -l) -eq 2 && $(minikube status | grep -q 'Correctly Configured')
-      meet: minikube start --vm-driver=xhyve --kubernetes-version=v1.7.5
+      meet: minikube start --kubernetes-version=v1.9.6 --vm-driver=hyperkit
```

Comment:
How would you feel about 1.9.4? Its the most recent version supported by minikube v0.25.2 see: `minikube get-k8s-versions`.  When using minikube v0.26.0 the most recent version I hit https://github.com/kubernetes/kubeadm/issues/733